pennylane_lightning/core/_serialize.py

```diff
@@ -441,18 +444,20 @@ def get_wires(operation, single_op):
                     controlled_wires_list,
                     controlled_values_list,
                 ) = get_wires(operation, single_op)
-                names.append(name)
+                inverses.append(inverse)
+                names.append(single_op.base.name if inverse else name)
                 # QubitUnitary is a special case, it has a parameter which is not differentiable.
                 # We thus pass a dummy 0.0 parameter which will not be referenced
                 if isinstance(single_op, qml.QubitUnitary):
                     params.append([0.0])
                     mats.append(matrix(single_op))
-                elif not hasattr(self.sv_type, name):
-                    params.append([])
-                    mats.append(matrix(single_op))
                 else:
-                    params.append(single_op.parameters)
-                    mats.append([])
+                    if hasattr(self.sv_type, single_op.base.name if inverse else name):
```

Comment:
Why nesting the `if` inside the `else` now instead of chaining an `elif` like before?

Reply:
This is mainly to reuse the logic for both inverse and named ops, while reducing the number of branches.